config.json

```diff
@@ -42,8 +42,8 @@
       "values": ["True", "False"],
       "defaultValue": true
     },
-    "TestArchitecture": {
-      "description": "Sets the architecture value that will be used for testing.",
+    "ArchitectureGroup": {
```

Comment:
@weshaggard did you want this to be passed in this way or embedded in the Configuration string?  I thought the latter.